extension/war/src/main/webapp/groovy/social/webui/space/UIManageAllSpaces.gtmpl

```diff
@@ -162,14 +163,9 @@ import org.exoplatform.social.core.service.LinkProvider;
 		          </h4>
 		          <div class="membersCount">$fieldLabelMembers <%= SpaceUtils.countMembers(space) %></div>
 		          <div class="content limitText" data-key="content" data-line="$maxLineDes">
-                    <p>
-                      <% if (space.description != null && !space.description.isEmpty()) { 
-                           print(space.description);
-                         } else { 
-                           print(_ctx.appRes("${uicomponent.id}.msg.default_space_description"));
-                         }
-                      %>
-                    </p>
+                    <% if (StringUtils.isNotBlank(space.description)) { %>
+                      <p href="<%=spaceUrl%>" data-placement="top" rel="tooltip" data-toggle="tooltip" data-original-title="$space.description"><%=space.description%></p>
```

Comment:
There is no attribute href for P HTML tag.

Reply:
done